### ISSUE TITLE
`ERROR: Parsing ../hockey-databank/AwardsPlayers.csv threw CSV error: record 2254 (line: 2255, byte: 82234): found record with 5 fields, but the previous record has 6 fields`

### ISSUE THREAD

**Issue by rippinrobr** [rippinrobr/csv-to#28]
Some records End with `wha,` instead of at the very least `wha,,`. I will have to see what I can do for hose rows and for characters like `ö`

_Originally posted by @rippinrobr in https://github.com/rippinrobr/csv-to/issues/21#issuecomment-438112934_

**Comment by rippinrobr** [comment]
I've added code to record the error message from the csv parser and then continue parsing the file.  Once the code is merged into master I will close this issue

**Comment by rippinrobr** [comment]
This could be done using something similar to the `retrosheet-events` parser.

```
let mut buf_reader = BufReader::new(file);
        let mut rdr = ReaderBuilder::new()
            .has_headers(false)
            .flexible(true) <----- specifically this
            .from_reader(buf_reader);
```

Now, if I'm to add this should I add a --flexible flag?
